### PR TITLE
Warning about the deprecation of primitives V1

### PR DESCRIPTION
**Pull request by 1ucian0**
It took me some time to find a warning that primitivesV1 is going to be removed. Making it more visible.

https://github.com/Qiskit/qiskit-ibm-runtime/pull/1798 is the api equivalent of this PR.

### CODE REVIEW

**Review by Eric-Arellano — APPROVED** [review]

File: docs/guides/primitives.mdx [hunk 1]
```diff
@@ -187,6 +193,12 @@ list2 = [SparsePauliOp("XX"), SparsePauliOp("XY"), SparsePauliOp("IZ")] # shape
 
 #### Sampler V1
 
+<Admonition type="warning">
+The [SamplerV1](/api/qiskit-ibm-runtime/qiskit_ibm_runtime.SamplerV1) has been deprecated in `qiskit-ibm-runtime 0.23`.
+Their support will be removed no earlier than July 15, 2024.
```
Comment (abbycross) [inline docs/guides/primitives.mdx:198]:
```suggestion
Their support will be removed no earlier than 15 July 2024.
```

File: docs/guides/primitives.mdx [hunk 2]
```diff
@@ -75,6 +75,12 @@ The updated interface uses a *primitive unified bloc* (PUB) for input.  Each PUB
 
 #### Estimator V1
 
+<Admonition type="warning">
+The [EstimatorV1](/api/qiskit-ibm-runtime/qiskit_ibm_runtime.EstimatorV1) has been deprecated in `qiskit-ibm-runtime 0.23`.
+Their support will be removed no earlier than July 15, 2024.
```
Comment (abbycross) [inline docs/guides/primitives.mdx:80]:
```suggestion
Their support will be removed no earlier than 15 July 2024.
```

File: docs/guides/primitives.mdx [hunk 3]
```diff
@@ -75,6 +75,12 @@ The updated interface uses a *primitive unified bloc* (PUB) for input.  Each PUB
 
 #### Estimator V1
 
+<Admonition type="warning">
+The [EstimatorV1](/api/qiskit-ibm-runtime/qiskit_ibm_runtime.EstimatorV1) has been deprecated in `qiskit-ibm-runtime` 0.23.
+Its support will be removed no earlier than 15 August 2024.
```
Comment (beckykd) [inline docs/guides/primitives.mdx:80]: Today Jessie told me that the date was moved to August 15.

File: docs/guides/primitives.mdx [hunk 4]
```diff
@@ -75,6 +75,12 @@ The updated interface uses a *primitive unified bloc* (PUB) for input.  Each PUB
 
 #### Estimator V1
 
+<Admonition type="warning">
+The [EstimatorV1](/api/qiskit-ibm-runtime/qiskit_ibm_runtime.EstimatorV1) has been deprecated in `qiskit-ibm-runtime` 0.23.
```
Comment (beckykd) [inline docs/guides/primitives.mdx:79]:
```suggestion
[EstimatorV1](/api/qiskit-ibm-runtime/qiskit_ibm_runtime.EstimatorV1) has been deprecated in `qiskit-ibm-runtime` 0.23.
```

File: docs/guides/primitives.mdx [hunk 5]
```diff
@@ -187,6 +193,12 @@ list2 = [SparsePauliOp("XX"), SparsePauliOp("XY"), SparsePauliOp("IZ")] # shape
 
 #### Sampler V1
 
+<Admonition type="warning">
+The [SamplerV1](/api/qiskit-ibm-runtime/qiskit_ibm_runtime.SamplerV1) has been deprecated in `qiskit-ibm-runtime` 0.23.
```
Comment (beckykd) [inline docs/guides/primitives.mdx:197]:
```suggestion
[SamplerV1](/api/qiskit-ibm-runtime/qiskit_ibm_runtime.SamplerV1) has been deprecated in `qiskit-ibm-runtime` 0.23.
```

**Review by beckykd — APPROVED** [review]
Approving with some small edits

File: docs/guides/primitives.mdx (hunk 1 above)
Comment (beckykd) [reply docs/guides/primitives.mdx]: I changed the date format, but Jessie said that the date was moved to August 15

**Comment by 1ucian0** [comment]
August 15 is, indeed, not earlier than July 15 :) The deprecation policy uses the formula "not earlier than X" to allow X+n to happen. However, if @jyu00 can confirm that the removal will be on August 15, I think the user will appreciate that extra precision. Once confirmed, I happy to replace   "will be removed no earlier than 15 July 2024" by "will be removed on 15 August 2024".

**Comment by jyu00** [comment]
> However, if @jyu00 can confirm that the removal will be on August 15,

Yeah the removal is planned for August 15.

**Comment by 1ucian0** [comment]
Change done in 7842ee9d96db7fddc7a4680aca83382199eddb0f with the confirmed removal date. Thanks!

**Review by frankharkins — APPROVED** [review]

**Pull request merged**